### DISCUSSION TITLE
Testers wanted for v10.0!

### DISCUSSION THREAD

**Discussion by folke** [LazyVim/LazyVim#1663] [Announcements]
I'm almost finished with a big rework that includes a bunch of new features.

Would be great if some people want to help me further test this new release.

For more details, see: https://github.com/LazyVim/LazyVim/pull/1655

Feel free to comment on that PR, or here with your feedback.

Thanks!

![image](https://github.com/LazyVim/LazyVim/assets/292349/cdb619d3-045b-4508-bf7f-a354e894568c)

**Comment by appelgriebsch** [comment]
Hey @folke thanks for the nice work on LazyVim and lazy.nvim. Really like using it on a daily basis. Do we still plan to introduce some kind of mechanism to enable extensions on users choice (similar to https://github.com/appelgriebsch/Nv/blob/main/init.lua#L14)? Would make it easier to choose which plugin to activate…

**Comment by abusch** [comment]
Any particular breaking changes to look out for when upgrading?

**Reply by abusch** [reply]
Ha, who am I kidding, I'll upgrade anyway...

**Reply by folke** [reply]
Some plugins were removed/replaced and some internal functions were renamed.

After upgrading, you shouldn't get any errors, just at most a bunch of deprecation warnings.

Do let me know if:

* any of the warnings aren't clear
* you get actual error messages instead
* something doesn't works as expected
* a new default makes little sense to you

Would also be great to test out the new `:LazyExtras`

**Reply by abusch** [reply]
Overall it was pretty smooth. I think the only error I had was because I had a spec for `none-ls` which isn't enabled by default anymore, so I had to enable the extra (haven't been bothered looking at `conform` yet...). I also commented out the different extras I had in my `config/lazy.lua` and re-enabled them via `LazyExtras` which worked perfectly (and is much easier!).

The only other thing I did I think was disable `dashboard.nvim` like I did for `alpha` :)

**Comment by dadav** [comment]
Having trouble getting nvim-lint to work. I tried to enable shellcheck for sh files like this:

```lua
  {
    "mfussenegger/nvim-lint",
    opts = {
      linters_by_ft = {
        sh = { "shellcheck" },
      },
    },
  },
  {
    "williamboman/mason.nvim",
    opts = {
      ensure_installed = {
        "shellcheck",
      },
    },
  },
```

But nothing is happening when I open shell scripts. No lsp is attached (LspInfo). Am I missing something?

**Reply by folke** [reply]
Nope, that's all that's needed. Shellcheck works for me.
There is no LSP with `nvim-lint`, so you won't see it in the list.

One thing to note (unrelated) is that your `ensure_installed` will override the whole config of mason since it's a list.

**Reply by dadav** [reply]
ok your are right..I was expecting an error when i run `echo $a` (without quotes). But somehow this is cool now :) Thanks folke.

**Reply by folke** [reply]
btw, you can also just use `bashls`, which includes linting with shellcheck

**Reply by dadav** [reply]
wow this is much better, thx again!
